etl/android-clients.kp/knowledge.md

```diff
@@ -161,11 +161,11 @@ while day <= end:
             StructField("defaultsearch", StringType(), True),
             StructField("device", StringType(), True),
             StructField("arch", StringType(), True),
-            StructField("fennecActivityStreamTopsitesLoaderTimeMs", 
+            StructField("fennec_activity_stream_topsites_loader_time_ms", 
```

Comment:
Should this PR also change `submissiondate` to `submission_date`, `creationdate` to `creation_date`, etc?